src/Microsoft.OData.Edm/Csdl/CsdlXmlWriterSettings.cs

```diff
@@ -0,0 +1,26 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="CsdlXmlWriterSettings.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.Edm.Csdl
+{
+    /// <summary>
+    /// The options/setting for CSDL writer
+    /// </summary>
+    public class CsdlXmlWriterSettings
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether the XML representation of the "Scale" attribute 
+        /// should use lowercase for the value "Variable".
+        /// </summary>
+        public bool UseLowercaseScaleVariable { get; set; }

```

Comment:
how about the CSDL JSON?

Reply:
For `CSDL JSON` we don't write the `variable` value. we write it like this:   `""@NS.DefaultDecimalTerm"": 0.34,`

Reply:
We currently do not write variable value for JSON.. I'd leave it out as a feature.